packages/components/src/systems/LocationSelect/index.tsx

```diff
@@ -49,7 +52,8 @@ export default function LocationSelect(props: LocationSelectProps) {
       const {
         coords: { latitude, longitude }
       } = await getNavigatorLocation({ timeout: 5000 });
-      const locations = await getLocations({ latitude, longitude }, geocoder!);
+      await asyncInterval(() => !!geocoder(), 10, 20);
+      const locations = await getLocations({ latitude, longitude }, geocoder()!);
```

Comment:
should (can?) we do the check for geocoder within getLocations?

Reply:
I have it as a param on getLocations so that it's not creating an instance of the geocoder each time it's called, could be a broader solution for this in the future though.